wikipedia_cleanup/predict.py

```diff
@@ -36,13 +42,19 @@ def __init__(
         self.timeframe_labels = ["day", "week", "month", "year"]
 
         self.predictor = predictor
-        own_relevant_attributes = ["value_valid_from"]
+        own_relevant_attributes = ["value_valid_from", "template"]
         self.relevant_attributes = list(
             set(own_relevant_attributes)
             | set(predictor.get_relevant_attributes())
             | set(self.group_key)
         )
         self.data: pd.DataFrame = pd.DataFrame()
+        self.run_results: Dict[str, Any] = {}
+        current_time = datetime.now()
+        self.run_id = (
```

Comment:
I would rather add an optional argument to the constructor and take the time format as a backup.